src/app/(main)/page.tsx

```diff
@@ -44,7 +45,11 @@ const Home = async () => {
   );
 
   if (!initialLandingPage) {
-    throw new Error("Page not found");
+    return <MissingData description={`Page for id '${landingId}'`} />
+  }
+
+  if (!initialLandingPage.data.sections) {
+    return <MissingData description={`Sections for landing page`} />
```

Comment:
for later: I think this should have the same welcome screen later so that the if would be ` if(!initialLandingPage || !initialLandingPage.data.sections)`